DEVDOCS-3807: Update checkout-sdk (#1457)
bigcommerce/dev-docs: docs/api-docs/cart-and-checkout/checkout-sdk.md

The Checkout JS SDK allows you to present the checkout process to the shopper in any way you like and, through front-end technologies and external web services, augment the checkout with extra information.

> However, the Checkout JS SDK does not allow you to change the checkout's underpinnings. You must still conform to the SDK's model of the Checkout API to complete a checkout and create an order in BigCommerce.

Reviewer: ```suggestion
However, the Checkout JS SDK does not allow you to change the checkout's underpinnings. You must still conform to the SDK's model of the Checkout API to complete a checkout and create an order in BigCommerce.

```

add a carriage return between paragraphs.

> However, the Checkout JS SDK does not allow you to change the checkout's underpinnings -- you must still conform to the model of the Checkout API underpinning the SDK to complete a Checkout and create an Order in BigCommerce.

Reviewer: ```suggestion
However, the Checkout JS SDK does not allow you to change the checkout's underpinnings. You must still conform to the SDK's model of the Checkout API to complete a checkout and create an order in BigCommerce.
```

## Where can I get the SDK?
The Checkout JS SDK and associated documentation is available from the [BigCommerce SDK Repo.](https://github.com/bigcommerce/checkout-sdk-js)

## How can I update the SDK?
The following steps describe how to update the Checkout SDK package in your existing custom checkout implementation.

> 1. Update the SDK by either running `npm install @bigcommerce/checkout-sdk@<latest-version-numer>` in your terminal, or

Reviewer: ```suggestion
1. Update the SDK in one of the two following ways:

* Run `npm install @bigcommerce/checkout-sdk@<latest-version-number>` in your terminal. 

```

> update the version number in your checkout's package.json file, then run npm install.

Reviewer: * Update the version number in your checkout implementation's package.json file, then run `npm install` in your terminal.

> <!-- checkout: info -->

Reviewer: ```suggestion
<!-- theme: info -->
```

> > Note: See [Changelog.md](https://github.com/bigcommerce/checkout-sdk-js/blob/master/CHANGELOG.md) for checkout-sdk version information.

Reviewer: ```suggestion
> #### Note 
> See the checkout-sdk-js repo's [changelog.md](https://github.com/bigcommerce/checkout-sdk-js/blob/master/CHANGELOG.md) for checkout-sdk version information.
```

## Where can I get the SDK?
The Checkout JS SDK and associated documentation is available from the [BigCommerce SDK Repo.](https://github.com/bigcommerce/checkout-sdk-js)

## How can I update the SDK?
The following steps describe how to update the Checkout SDK package in your existing custom checkout implementation.

1. Fork and clone [checkout-js](https://github.com/bigcommerce/checkout-js), then install its dependencies with the clean slate install command.

```shell title="Clone your fork and install dependencies"
# Clone the repo
git clone https://github.com/<your-user-name>/checkout-js checkout-js

cd checkout-js

# Install dependencies with a clean slate
npm ci
```

> 2. Open your store's `package.json` file in your text editor.

Reviewer 2: You can also go to the terminal and run `npm install @bigcommerce/checkout-sdk@<latest/version number>`

> 3. Update the value for "@bigcommerce/checkout-sdk". Enter the minimum version for the gateway to work. 

Reviewer 3: Think it's worth mentioning we provide checkout-sdk version information on [this page](https://github.com/bigcommerce/checkout-sdk-js/blob/master/CHANGELOG.md).

## How can I update the SDK?
The following steps describe how to update the Checkout SDK package in your existing custom checkout implementation.

1. In your terminal, navigate to the project root of your customer checkout implementation. You can update the SDK in one of the two following ways:

> * Run `npm install @bigcommerce/checkout-sdk@<latest-version-number>` in the terminal. 

Reviewer: ```suggestion
* Run `npm install @bigcommerce/checkout-sdk@<latest-version-number>` in the terminal; or 
```
